src/lolex-src.js

```diff
@@ -193,6 +191,24 @@ function withGlobal(_global) {
             }
         }
 
+        function ClockDate() {
+            var args = Array.prototype.slice.call(arguments);
+
+            if (args.length === 0) {
+                args.push(ClockDate.clock.now);
+            }
+
+            var date = createNativeDate.apply(this, args);
+
+            // if Date is called as a constructor with 'new' keyword
+            if (this instanceof ClockDate) {
+                return date;
+            }
+
+            // if Date called as function - ecma-262/#sec-date-year-month-date-hours-minutes-seconds-ms
```

Comment:
Since we are actually explicitly targetting version 5.1, it's worth adding that this has remained unchanged from version 5.1, where it is mentioned in [section 15.9.2](https://www.ecma-international.org/publications/files/ECMA-ST-ARCH/ECMA-262%205.1%20edition%20June%202011.pdf).
```suggestion
            // the Date constructor called as a function, ref Ecma-262 Edition 5.1, section 15.9.2. 
            // This remains so in the 10th edition of 2019 as well.
```

Reply:
Done